PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+## Description
```

Comment:
Could these be questions?
e.g.
"What are you trying to achieve?"
"What does it look like?"
"Should have you added: tests..?"